components/Downloads/DownloadCards/index.tsx

```diff
@@ -0,0 +1,82 @@
+import { Tabs, TabList } from 'react-tabs';
+import { useState, useEffect } from 'react';
+import styles from './index.module.scss';
+import DownloadCard from './DownloadCard/index';
+import type { FC, KeyboardEvent } from 'react';
+import type { UserOS } from '../../../types/userOS';
+
+type DownloadCardsProps = {
+  version: string;
+  userOS: UserOS;
+};
+
+const DownloadCards: FC<DownloadCardsProps> = ({ version, userOS }) => {
+  const [selected, setSelected] = useState('');
+
+  useEffect(() => {
+    setSelected(!['WIN', 'MAC'].includes(userOS) ? 'SOURCECODE' : userOS);
+  }, [userOS]);
+
+  const downloadTypes = [
+    {
+      name: 'WIN',
+      icon: 'windows-download-logo.svg',
+      label: 'Windows Installer',
```

Comment:
We should use an translation key

For example in French we will write. "Installateur Windows" instead of Windows Installer

Reply:
I think it is a good suggestion, thanks!